cli/util/util.go

```diff
@@ -62,10 +62,14 @@ func AskYesNo(question string) bool {
 
 }
 
-func GetCurrentDockerVersion() *version.Version {
+func GetRawCurrentDockerVersion() string {
 	output, _ := exec.Command("docker", "--version").Output()
-	re := regexp.MustCompile("Docker version ([\\d|\\.]+)")
-	versionNumber := re.FindAllStringSubmatch(string(output), -1)[0][1]
+	re := regexp.MustCompile("Docker version (.*),")
```

Comment:
The format is "Docker version [theversionnumber], build [buildhash]" so feel comfortable (for now) grabbing everything before the space.